Userland/Libraries/LibWeb/Bindings/IDLAbstractOperations.h

```diff
@@ -120,64 +120,20 @@ JS::Completion call_user_object_operation(Bindings::CallbackType& callback, Stri
     return clean_up_on_return(stored_settings, relevant_settings, completion);
 }
 
+// https://webidl.spec.whatwg.org/#invoke-a-callback-function
```

Comment:
aha, here it is - would rather have it next to the implementation instead of the declaration.

Reply:
fixed